src/main/java/hudson/plugins/gradle/injection/GradleEnterpriseExceptionLogFilter.java

```diff
@@ -14,7 +14,7 @@ public class GradleEnterpriseExceptionLogFilter extends ConsoleLogFilter impleme
     @Override
     public OutputStream decorateLogger(Run build, OutputStream logger) {
         InjectionConfig injectionConfig = InjectionConfig.get();
-        if (injectionConfig.isEnabled() && injectionConfig.isCheckForBuildAgentErrors()) {
+        if (injectionConfig.isEnabled() && injectionConfig.isCheckForBuildAgentErrors() && build != null) {
```

Comment:
[Probably unnecessary](https://github.com/jenkinsci/workflow-api-plugin/blob/41b0279c361a2ee1d0435e7304b55c3532bcecc9/src/main/java/org/jenkinsci/plugins/workflow/log/TaskListenerDecorator.java#L149-L151), but core method does not specify nullability of the `build` argument, so being defensive.